Content/UseSbieDeskHack.md

```diff
@@ -0,0 +1,29 @@
+# Use SbieDesk Hack
+
+_UseSbieDeskHack_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md).
+
+```
+   .
+   .
+   .
+   [DefaultBox]
+   UseSbieDeskHack=y
+```
+
+A desktop object solution that is now enabled by default for all processes.
+
+
+**Technical Details**
+
+This is a desktop object solution that is used for all processes. 
+
+It was initially implemented to address the issue of infinite callback problems caused by delayed loading (the infinite recursion problem has been resolved in version 5.43). 
```

Comment:
> (the infinite recursion problem has been resolved in version 5.43).

I know you took this info from [here](https://github.com/sandboxie-plus/Sandboxie/commit/43d61c50bc0420790f13dc4b79deb7a2b27f717d#diff-55d44deb107f22cc4864816fe46df3f3e64fa82aa00af5fd69d270cbe70d36f7), but could you point me to the lines that introduced the recursion fix? The note you refer to was written some time after the fix, so I would like to be sure it is accurate.

Reply:
At that time, I examined the relevant code and ultimately located it in the `Build 0.4.0 / 5.43` submission of version `v0.3...v0.4.0` on [Github/Sandboxie/v0.3...v0.4.0#diff](https://github.com/sandboxie-plus/Sandboxie/compare/v0.3...v0.4.0#diff-60bf43005a5ae1c852835126db27eabf6130844c67f51abad8c8d471c4c791e8).